assignments/callbacks.js

```diff
@@ -24,24 +24,57 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr)
 }
+function lengthOfArr(arr) {
+  return arr.length
+}
+console.log(getLength(items, lengthOfArr));
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr)
+}
+function lastItem(arr) {
+  return arr.pop();
 }
+console.log(last(items, lastItem))
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x,y);
```

Comment:
remember, you're doing the operator and then passing it into the cb function:

cb(x + y);